config-linux.md

```diff
@@ -494,6 +494,21 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
```

Comment:
```markdown
On cgroup v2 hosts, if both unified and legacy fields are set, the OCI runtime SHOULD apply both. 
In case of a conflict across the unified field and the legacy fields, the OCI runtime SHOULD error out.
```

Reply:
I've added a new section for the "conversion" part

Reply:
`to be set and modified for the`

Reply:
thanks, fixed